src/shared/lib/validation/user.ts

```diff
@@ -15,16 +15,16 @@ export function validateNotificationsOn(notificationsOn: boolean): Validation<Da
   return notificationsOn ? valid(new Date()) : valid(null);
 }
 
-export function validateAcceptedTerms(acceptedTerms: boolean): Validation<Date> {
-  return acceptedTerms ? valid(new Date()) : invalid(['You cannot unaccept the terms and conditions.']);
+export function validateAdminStatus(adminStatus: boolean): Validation<UserType> {
+  return adminStatus ? valid(UserType.Admin) : valid(UserType.Government);
```

Comment:
I don't think this needs to be a validation since it takes `boolean` and only returns valid values. My suggestion is to simply create a helper function that converts a boolean to `UserType`. Something like `adminPermissionsToUserType(admin: boolean): UserType`

Reply:
Yes, I will do this.

Reply:
Done, also switched validation of notifications flag to helper.